subsys/bluetooth/controller/ll_sw/ull.c

```diff
@@ -1827,6 +1827,18 @@ void ull_prepare_dequeue(uint8_t caller_id)
 	}
 }
 
+struct event_done_extra *ull_event_done_extra_dequeue(void)
+{
+	struct node_rx_event_done *evdone;
+
+	evdone = MFIFO_DEQUEUE_PEEK(done);
+	if (!evdone) {
+		return NULL;
+	}
+
+	return &evdone->extra;
+}
+
```

Comment:
Isn't this duplicate of `ull_event_done_extra_get()` ?
```suggestion
```

Reply:
Ah.. sorry it was left over from previous iteration of the change